crates/pet-core/src/os_environment.rs

```diff
@@ -133,8 +133,18 @@ impl Environment for EnvironmentApi {
     }
 }
 
+#[cfg(windows)]
+fn get_user_home() -> Option<PathBuf> {
+    let home = env::var("USERPROFILE").or_else(|_| env::var("HOME"));
```

Comment:
This was the problem, on windows, we need to use USERPROFILE instead of HOME.

Previously was using HOME over USERPROFILE with the assumption that win users will not have HOME var set. that was clearly wrong.